e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishSingleNodeStandaloneTestTheory.cs

```diff
@@ -18,59 +19,33 @@ namespace IIoTPlatform_E2E_Tests.Standalone {
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Standalone Test Collection")]
     [Trait(TestConstants.TraitConstants.PublisherModeTraitName, TestConstants.TraitConstants.PublisherModeStandaloneTraitValue)]
-    public class PublishNodesFromFileTestTheory {
+    public class PublishSingleNodeStandaloneTestTheory {
```

Comment:
SwitchToStandalone mode in the beginning and SwitchToOrchestratedMode in the end are missing

Reply:
No, SwitchToOrchestrated is hapenning in the orchestrated test. SwitchToStandalone in the Start Publishing test as the deployments must be updated first.